docs/create_development_environment.md

```diff
@@ -33,7 +33,7 @@ The above will generate a `package.json` file that looks as follow:
 
 ```
 
-In order to wrap our multi file project into a single file, we will use [webpack](https://webpack.js.org/), let install the required webpack tools:
+In order to wrap our multi-file project into a single file, we will use [webpack](https://webpack.js.org/), let install the required webpack tools:
```

Comment:
To wrap our multi-file project into a single file, we will use [webpack](https://webpack.js.org/), and install the required webpack tools: